pkg/broker/broker.go

```diff
@@ -29,6 +29,13 @@ var (
 	ErrorBindingExists = errors.New("binding exists")
 )
 
+const (
+	// provisionCredentialsKey - Key used to pass credentials to apb.
+	provisionCredentialsKey = "_apb_provision_creds"
+	// bindCredentialsKey - Key used to pas bind credentials to apb.
+	bindCredentialsKey = "_apb_bind_creds"
```

Comment:
These are more descriptive but make for longer key names when used as a map key. 

```
params[provisionCredentialsKey]
```

vs
```
params[provisionCredsKey]
```

I guess it's fine.